bin/build-frankenphp freetype: unexpected token `win32-dll'

```console
  configuration directory     ./builds/unix
  configuration rules         ./builds/unix/unix.mk

If this does not correspond to your system or settings please remove the file
`config.mk' from this directory then read the INSTALL file for help.

Otherwise, simply type `make' again to build the library,
or `make refdoc' to build the API reference (this needs Python >= 3.5).

cd builds/unix; \
        ./configure
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking how to run the C preprocessor... gcc -E
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.24... yes
./configure: line 3463: syntax error near unexpected token `win32-dll'
./configure: line 3463: `LT_INIT(win32-dll)'
make: *** [builds/unix/detect.mk:91: setup] Error 2
```

I've tried upgrading libtool and other shenanigans, but this still happens. Manually removing the win32-dll target doesn't solve it either because it then can't find ./build/unix/libtool (which shouldn't be there anyhow).

--- a/src/SPC/builder/unix/library/freetype.php
+++ b/src/SPC/builder/unix/library/freetype.php
@@ -18,20 +18,21 @@ trait freetype
      */
     protected function build(): void
     {
-        $extra_libs = $this->builder->getLib('libpng') ? '--with-png' : '--without-png';
-        $extra_libs .= ' ';
-        $extra_libs .= $this->builder->getLib('bzip2') ? ('--with-bzip2=' . BUILD_ROOT_PATH) : '--without-bzip2';
-        $extra_libs .= ' ';
-        $extra_libs .= $this->builder->getLib('brotli') ? ('--with-brotli=' . BUILD_ROOT_PATH) : '--without-brotli';
-        $extra_libs .= ' ';
-
-        shell()->cd($this->source_dir)
+        $extra_libs = $this->builder->getLib('libpng') ? '-DFT_DISABLE_PNG=OFF ' : '-DFT_DISABLE_PNG=ON ';
+        $extra_libs .= $this->builder->getLib('bzip2') ? '-DFT_DISABLE_BZIP2=OFF ' : '-DFT_DISABLE_BZIP2=ON ';
+        $extra_libs .= $this->builder->getLib('brotli') ? '-DFT_DISABLE_BROTLI=OFF ' : '-DFT_DISABLE_BROTLI=ON ';
+        FileSystem::resetDir($this->source_dir . '/build');
+        shell()->cd($this->source_dir . '/build')
             ->setEnv(['CFLAGS' => $this->getLibExtraCFlags(), 'LDFLAGS' => $this->getLibExtraLdFlags(), 'LIBS' => $this->getLibExtraLibs()])
-            ->execWithEnv('./autogen.sh')
-            ->execWithEnv('./configure --without-harfbuzz --prefix= ' . $extra_libs)
+            ->execWithEnv(
+                "cmake {$this->builder->makeCmakeArgs()} -DFT_DISABLE_HARFBUZZ=ON " .
+                '-DBUILD_SHARED_LIBS=OFF ' .
+                "{$extra_libs}.."
+            )
             ->execWithEnv('make clean')
             ->execWithEnv("make -j{$this->builder->concurrency}")
-            ->execWithEnv('make install DESTDIR=' . BUILD_ROOT_PATH);
+            ->execWithEnv('make install');
+
         $this->patchPkgconfPrefix(['freetype2.pc']);
         FileSystem::replaceFileStr(
             BUILD_ROOT_PATH . '/lib/pkgconfig/freetype2.pc',

--- a/src/globals/test-extensions.php
+++ b/src/globals/test-extensions.php
@@ -36,17 +36,17 @@ $no_strip = false;
 $upx = false;
 
 // prefer downloading pre-built packages to speed up the build process
-$prefer_pre_built = true;
+$prefer_pre_built = false;
 
 // If you want to test your added extensions and libs, add below (comma separated, example `bcmath,openssl`).
 $extensions = match (PHP_OS_FAMILY) {
-    'Linux', 'Darwin' => 'grpc',
+    'Linux', 'Darwin' => 'gd',
     'Windows' => 'bcmath',
 };
 
 // If you want to test lib-suggests feature with extension, add them below (comma separated, example `libwebp,libavif`).
 $with_libs = match (PHP_OS_FAMILY) {
-    'Linux', 'Darwin' => '',
+    'Linux', 'Darwin' => 'freetype',
     'Windows' => '',
 };
 
@@ -54,7 +54,7 @@ $with_libs = match (PHP_OS_FAMILY) {
 // You can use `common`, `bulk`, `minimal` or `none`.
 // note: combination is only available for *nix platform. Windows must use `none` combination
 $base_combination = match (PHP_OS_FAMILY) {
-    'Linux', 'Darwin' => 'none',
+    'Linux', 'Darwin' => 'minimal',
     'Windows' => 'none',
 };
 

